src/FilterConditionSelector/FilterConditionSelector.tsx

```diff
@@ -46,7 +46,14 @@ const FilterConditionSelector: React.FC<
 	const listRef = React.createRef<HTMLUListElement>();
 	const mainRef = React.useRef<HTMLElement | null>(null);
 	const [searchTerm, setSearchTerm] = React.useState(props.searchTerm);
+	const [
+		firstItemTree,
```

Comment:
Это дерево, которое является первым элементом чего-то или первый элемент дерева?
Может
```suggestion
		treeFirstItem,
```